### PR TITLE
Release version 2.1.8

### PR DESCRIPTION
**Pull request by stel**
Because of a lot of users are affected by #297 I think it's better to make a new release asap from a separate cocoa branch that contains https://github.com/realm/realm-object-store/pull/450.

### CODE REVIEW

File: RealmBrowserTests/Supporting Files/RealmBrowserTests-Info.plist [hunk 1]
```diff
@@ -13,10 +13,10 @@
 	<key>CFBundlePackageType</key>
 	<string>BNDL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>2.1.8</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>92</string>
```
Comment (stel) [inline RealmBrowserTests/Supporting Files/RealmBrowserTests-Info.plist]: I started using agvtool (inside fastlane) which modifies version number for all targets in projects. Actually having the same version for tests target seems absolutely fine.

**Comment by FreudGit** [comment]
Hope will be release today.  My realm browser crash at db open. :(

Any chance?

**Comment by stel** [comment]
@FreudGit we're experiencing some issues with our CI currently. If it's urgent case I suggest you to build the Browser by yourself from this brunch https://github.com/realm/realm-browser-osx/tree/do-release-2.1.8.

**Comment by stel** [comment]
@toanz I've replied in https://github.com/realm/realm-browser-osx/issues/297.

The build has been sent for Apple Review and will be release right after it's approved.

**Comment by stel** [comment]
2.1.8 is available on the Mac App Store.

**Pull request merged**